DistributionUpdate/PackageUpdate/PackageInfoTools.g

```diff
@@ -1591,6 +1591,15 @@ AddHTMLPackageInfo := function(arg)
   # SuggestUpgrades  entry
   info.SuggestUpgradesEntry := Concatenation("[ \"", info.PackageName,
        "\", \"", info.Version, "\" ], ");
+
+  # license ...
+  if IsBound(info.License) then
+    Append(res, Concatenation("<h4>License</h4>\n<p>",
+                   "<p><a href='https://spdx.org/licenses/",
```

Comment:
Actually, this is too simplistic in general, it can't deal with complex license constructs involving operators like `AND`, `OR` -- see also <https://spdx.org/spdx-specification-21-web-version>

A few packages use these (but AFAIK none uses `WITH`). Specifically, `matrgrp` has `GPL-2.0 OR GPL-3.0`, while `profiling` has `MIT AND CDDL-1.0` as license.

I guess I could implement a function that deconstructs a "complex" SPDX license string, and use it here and in GitHubPagesForGAP. Though I guess at least initially, I'd restrict it at first to only support pure disjunctions and conjunctions, and no parenthesis.

Reply:
Despite my comment here, I think we could merge this as is -- right now, exactly two packages would have broken license links, and I think that's acceptable. Or, if you disagree, we could just handle those two special cases.
In any case, it would be really helpful if this change gets tested during the next distribution release; it's much easier to refine something that is known to at least work rudimentary, than to flail around with completely untested code. (That said, of course best would be if I have a clue how to test this code in the first place :-( ).

Reply:
ok, should be doable when GAP 4.10.2 comes shortly.